src/dummygamedef.h

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
These dummy files are used by the benchmark, so it might make sense to put them there?